src/encoded/schemas/transgenic_enhancer_experiment.json

```diff
@@ -0,0 +1,269 @@
+            "comment": "Do not submit. date_submitted will be assigned only by DCC personnel.",
+            "type": "string",
+            "format": "date",
+            "permission": "import_items"
+        },
+        "dbxrefs": {
+            "@type": "@id",
+            "rdfs:subPropertyOf": "rdfs:seeAlso",
+            "title": "External resources",
+            "description": "Identifiers from external resources that may have 1-to-1 or 1-to-many relationships with ENCODE objects.",
+            "type": "array",
+            "default": [],
+            "uniqueItems": true,
+            "items": {
+                "title": "External identifier",
+                "description": "An identifier from an external resource that may have 1-to-1 or 1-to-many relationships with ENCODE objects.",
+                "type":  "string",
+                "pattern": "^(GEO:(GSM|GSE)\\d+|VISTA:(hs|mm)\\d+)$"
+            }
+        },
+        "documents": {
+            "description": "Protocols or other documents that describe the assay or the results (not data files)."
+        },
+        "elements_references": {
+            "title": "Elements references",
+            "description": "Datasets that contain reference files listing the elements being characterized in this functional characterization experiment.",
+            "type": "array",
+            "default": [],
+            "uniqueItems": true,
+            "items": {
+                "title": "Reference",
+                "description": "Dataset that contain reference file or annotation listing the elements being characterized in this functional characterization experiment.",
+                "comment": "See reference.json or annotation.json for a list of available identifiers.",
+                "type": "string",
+                "linkTo": ["Reference", "Annotation"]
+            }
+        },
+        "biosamples": {
+            "title": "Biosamples",
+            "description": "The biosamples used in the transgenic enhancer experiment.",
+            "type": "array",
+            "uniqueItems": true,
+            "minItems": 1,
+            "items": {
+                "title": "Biosample",
+                "description": "The unique test organism used in the transgenic enhancer experiment.",
+                "comment": "See biosample.json for a list of available identifiers.",
+                "type": "string",
+                "linkTo": "Biosample"
+            }
+        },
+        "tissue_with_enhancer_activity": {
+            "title": "Tissues with enhancer activity",
+            "description": "The tissue(s) in which enhancer activity was positively expressed in.",
+            "type": "array",
+            "uniqueItems": true,
+            "minItems": 1,
+            "items": {
+                "title": "Tissue",
+                "description": "Tissue where positive expression of the enhancer was observed.",
+                "type": "string",
```

Comment:
I would re-word to be something like "The tissue in which enhancer activity was detected"

Reply:
Changed wording accordingly